### ISSUE TITLE
Docs website missing changelog older than 19.6.2

### ISSUE THREAD

**Issue by markerikson** [crossbario/crossbar#1645]
The Changelog page at https://crossbar.io/docs/changelog/ only covers releases since 19.6.2 .  I see there is also an older changelog file at https://github.com/crossbario/crossbar/blob/86e4d93251d25d430f6bc223a85c412ac52ace7a/docs-old/pages/ChangeLog.md , but it appears to only cover up to 18.3.1.  That file's contents are not included in the website, and there's no list of changes for versions between 18.3.1 and 19.6.2.

**Comment by oberstet** [comment]
yes, unfortunately. but won't fix. why do you need old changelogs?

> the new manually maintained changelog on the website/docs comes from https://github.com/crossbario/crossbar/blob/master/docs/changelog.rst

**Comment by markerikson** [comment]
We're currently running Crossbar 17.6.1.  We're loading some custom "components", and also have a couple hand-patches being applied.  A coworker was trying to apply some config changes and it looks like a number of config options have changed or been added (such as being able to specify SSL cert config for components).  We clearly need to upgrade, but we need to know what's actually changed in the intervening versions to know if there's anything that affects us.
